lib/index.js

```diff
@@ -198,7 +199,7 @@ function getExperiments(options) {
     if (experiment) {
       results.push({
         variationName: options.variationNamesMap[experimentId],
-        variationId: options.variationIdsMap[experimentId][0],
+        variationId: options.variationIdsMap[experimentId][0].toString(),
```

Comment:
Cool sounds good!

@wcjohnson11 In the multivariate case, could we send an array of numbers (and in the single case, array of a single number) instead of strings? Any reason why were sending a comma separated string?